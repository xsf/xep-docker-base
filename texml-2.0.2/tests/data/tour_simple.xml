<TeXML>
<TeXML escape="0">
\documentclass[a4paper]{article}
\usepackage[latin1]{inputenc}
\usepackage[T1]{fontenc}
</TeXML>
<env name="document">
I'm not afraid of the symbols
$, ^, > and others.
</env>
</TeXML>
