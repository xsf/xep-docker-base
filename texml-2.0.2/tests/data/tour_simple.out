\documentclass[a4paper]{article}
\usepackage[latin1]{inputenc}
\usepackage[T1]{fontenc}
\begin{document}
I'm not afraid of the symbols
\textdollar{}, \^{}, \textgreater{} and others.
\end{document}
